prow/cmd/branchprotector/BUILD

```diff
@@ -1,3 +1,42 @@
+# Usage:
+#   bazel run :dev-job.{create,delete,describe}
+load("@io_bazel_rules_k8s//k8s:object.bzl", "k8s_object")
+
+k8s_object(
+    name = "dev-job",
+    cluster = "{STABLE_PROW_CLUSTER}",  # Run make :get-cluster-credentials
+    image_chroot = "{STABLE_DOCKER_REPO}",
+    images = {
+        "gcr.io/k8s-testimages/branchprotector:latest": ":branchprotector-image",
+    },
+    kind = "Job",
+    template = ":dev-job.yaml",
```

Comment:
I'm wondering if "oneshot" would be a better qualifier than "dev-"?

Reply:
Done